src/useStyles.js

```diff
@@ -2,20 +2,22 @@ import { makeStyles } from '@material-ui/core/styles'
 
 const useStyles = (borderOptions) => makeStyles((theme) => {
   const containerBorderStyle = borderOptions.isBorder === true && {
-    border: `${borderOptions.borderStyle.size}px ${borderOptions.borderStyle.type} ${borderOptions.borderStyle.color ? borderOptions.borderStyle.color : 'black'}`,
-    'border-radius': `${borderOptions.borderStyle.radius > 0 ? borderOptions.borderStyle.radius : 0}px`,
+    borderWidth: `${borderOptions.borderStyles.borderWidth ? borderOptions.borderStyles.borderWidth : 2}px`,
+    borderStyle: `${borderOptions.borderStyles.borderStyle ? borderOptions.borderStyles.borderStyle : 'solid'}`,
+    borderColor: `${borderOptions.borderStyles.borderColor ? borderOptions.borderStyles.borderColor : 'black'}`,
+    borderRadius: `${borderOptions.borderStyles.borderRadius > 0 ? borderOptions.borderStyles.borderRadius : 0}px`,
```

Comment:
Do we need conditional statements here when `borderStyles`'s default values are set by PropTypes?

Reply:
also let's set default border style to :
```css
border: 1px solid #e0e0e0
border-radius: 4px

Reply:
problem is when the user pass a value lets say only border-radius the default values won't take effect only the passed border-radius